proto/iotics/api/meta.proto

```diff
@@ -38,7 +36,7 @@ service MetaAPI {
   rpc SparqlUpdate(SparqlUpdateRequest) returns (SparqlUpdateResponse) {}
 
   // ExplorerQuery - Deprecated - use SparqlQuery instead.
-  rpc ExplorerQuery(ExplorerRequest) returns (stream SparqlQueryResponse) {}
+  rpc ExplorerQuery(ExplorerRequest) returns (stream ExplorerQueryResponse) {}
```

Comment:
Breaking change affecting portal

Reply:
Mistake - I'll remove `Query` from this (since it's **not** `ExplorerQueryRequest`)

Reply:
24cd4a2